core/src/main/java/org/bitcoinj/core/FeeFilterMessage.java

```diff
@@ -47,7 +47,7 @@ protected void bitcoinSerializeToStream(OutputStream stream) throws IOException
 
     @Override
     protected void parse() throws ProtocolException {
-        feeRate = Coin.ofSat(readUint64().longValue());
+        feeRate = Coin.read(payload);
```

Comment:
I would go with: `Coin.ofSat(readInt64())` and not add the `ByteBuffer` stuff to `Coin`.